docs/modules/reference/pages/configuration/tuning-kafka.adoc

```diff
@@ -32,7 +32,7 @@ org.opennms.instance.id=MyNMS
 == Tips for Kafka
 
 * For Kafka RPC, the number of partitions should always be greater than the number of Minions at a location.
-When there are multiple locations, partitions >= max number of minions at a location.
+When there are multiple locations, the partition count should be greater than the max number of Minions at a location.
```

Comment:
The recommendation for the number of partitions should be a multiple of the number of minions in a particular location.

This will allow for a more even distribution of work.

Reply:
```suggestion
When there are multiple locations, the partition count should be based on the maximum number of Minions at a location.
```

Reply:
So generally a location gets it's own topic(s), thus you may want to amend it to something like.

"When there are multiple locations, each location will have its own topics. The partition count for all topics assigned to a particular location should be based on a multiple of the maximum number of Minions assigned to that location."

Thoughts?

Reply:
```suggestion
When there are multiple locations, the partition count for RPC topics should be based on a multiple of the maximum number of Minions at the associated location.
```